fundrebalancer/fundrebalancer.go

```diff
@@ -263,13 +263,23 @@ func (r *FundRebalancer) MoveFundsToChain(
 		}
 		txn := txns[0]
 
+		approvalHash, err := r.ERC20Approval(ctx, txn)
```

Comment:
why do we not include the erc20 approval in the gas cost estimation?

Reply:
not sure, should probably add that in a follow up pr